bin/apisix

```diff
@@ -260,7 +266,12 @@ http {
     init_by_lua_block {
         require "resty.core"
         apisix = require("apisix")
-        apisix.http_init()
+
+        local dns_resolver = { {% for _, dns_addr in ipairs(dns_resolver or {}) do %} "{*dns_addr*}", {% end %} }
+        local args = {
+            dns_resolver = dns_resolver,
+        }
+        apisix.http_init(args)
```

Comment:
we need to passed `args` to stream case

https://github.com/apache/incubator-apisix/blob/master/bin/apisix#L129-L133

Reply:
https://github.com/apache/incubator-apisix/pull/1217#discussion_r391652590

stream not need dns resolver, so removed.